### DISCUSSION TITLE
Math expressions not rendered when MathJax renderer set SVG

### DISCUSSION THREAD

**Discussion by tamuratak** [community/community#17068] [Other Feature Feedback, Questions, & Ideas]
1. Set MathJax renderer as SVG
2. Edit math expressions
3. Cannot render them

The setting menu `Math Renderer` should be removed at least.

![mathjaxrenderer](https://user-images.githubusercontent.com/10665499/169673375-606b9363-9119-442a-96c9-f23406f02e05.png)

![mathexp](https://user-images.githubusercontent.com/10665499/169673379-3fbd1d89-7228-4f07-bc4f-f456427503b7.png)

**Comment by fortunewalla** [comment]
$$a+gb+c$$

> 1. Set MathJax renderer as SVG
> 
>     2. Edit math expressions
> 
>     3. Cannot render them
> 
> 
> The setting menu `Math Renderer` should be removed at least.
> 

It does render it but one needs to refresh the page. After refresh, your example equation renders proper and also the setting remains in SVG mode. I'm using Firefox BTW.

**Comment by tamuratak** [comment]
> It does render it but one needs to refresh the page. 

Yes, but rendered in CHTML mode, not SVG mode.

![mathjaxchtml](https://user-images.githubusercontent.com/10665499/169680849-9bc74d38-502e-47db-a99d-0a9c85c19b37.png)
